src/components/NcAvatar/NcAvatar.vue

```diff
@@ -374,7 +387,12 @@ export default {
 			if (this.ariaLabel !== null) {
 				return this.ariaLabel
 			}
-
+			if (this.title) {
+				if (this.hasStatus && this.showUserStatus && this.showUserStatusCompact) {
+					return t('Avatar of {title}, {status}', { title: this.title ?? this.user, status: this.userStatus.status })
+				}
+				return t('Avatar of {title}', { title: this.title ?? this.user })
```

Comment:
`this.title` will always be defined here because you exclude this if block by checking if `this.title` is defined

maybe set the fallback to user display name earlier

please double check if `this.user` is the display name and not the user id

Reply:
this.user is the display name and not the user id

Reply:
> this.title will always be defined here because you exclude this if block by checking if this.title is defined

make sense. I will leave comp prop ```avatarAriaLabel``` as it already was, just added a small tooltip (in form of ```title```) to ```<div>```. Detailed description will be found in ```aria-label```